src/server/cluster_family_test.cc

```diff
@@ -33,8 +33,47 @@ class ClusterFamilyTest : public BaseFamilyTest {
   static constexpr string_view kInvalidConfiguration = "Invalid cluster configuration";
 };
 
+TEST_F(ClusterFamilyTest, DflyClusterOnlyOnAdminPort) {
+  EXPECT_EQ(RunAdmin({"dflycluster", "config", R"json(
+      [
+        {
+          "slot_ranges": [
+            {
+              "start": 0,
+              "end": 16383
+            }
+          ],
+          "master": {
+            "id": "abcd1234",
+            "ip": "10.0.0.1",
+            "port": 7000
+          },
+          "replicas": []
+        }
+      ])json"}),
+            "OK");
```

Comment:
Specifically for this test, I'd set this string to a variable so it's easier to see that it's exactly the same string